app/controllers/interviews_controller.rb

```diff
@@ -0,0 +1,74 @@
+class InterviewsController < ApplicationController
+	def index
+	  @interviews = Interview.order('created_at DESC')
+	end
+
+	def new
+	  @interview= Interview.new
+	  @users=User.all
+      @selected=Array.new
```

Comment:
Why do you need this

Reply:
users is for the display of the available email ids for selection
interview is for the display of errors
and selected has been just initialized to new as none of the users will be selected before the display of new form.